contrib/ohi-release-notes/action.yml

```diff
@@ -0,0 +1,94 @@
+name: OHI release notes
+description: Wrapper for release toolkit that runs commands needed to release an OHI
+inputs:
+  git-root:
+    description: Path to the root of the git repository to source bot commits from.
+    default: "."
+  excluded-dirs:
+    description: Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".
+    default: '.github'
+  fail-if-held:
+    description: Fail if the held toggle is active. Defaults to `true`.
+    default: 'true'
+  link-dependencies-dictionary:
+    description: Sets the link dependency dictionary. Defaults to ".github/rt-dictionary.yaml".
+    default: ''
```

Comment:
Maybe there is some magic there and I'm wrong, but I think the

> Defaults to ".github/rt-dictionary.yaml"

and the corresponding readme comment

don't match with the `default: ''`

Reply:
It used to be that default but I had to change it and forgot to remove it from the description.

Fixed in [#345ad7a](https://github.com/newrelic/release-toolkit/pull/135/commits/345ad7a05cd21e2d2aaf0f3f46b423161917de2e)